vars/manifest.yml

```diff
@@ -12,3 +12,18 @@ upstream_repos:
   loci:
     src: https://github.com/openstack/loci
     version: c94e1dfc86cae044a4582f321bb1721846419a00
+  airship-armada:
+    src: https://github.com/openstack/airship-armada
+    version: master
```

Comment:
you should probably freeze this -- but that's your business.

Reply:
Agreed, we will have to do that. I think, current thinking was that as we are still in development mode so don't want to miss out fixes happening in upstream for problems we may see or already observed.

Reply:
Fine for me :)